src/seedu/addressbook/data/person/Block.java

```diff
@@ -0,0 +1,13 @@
+package seedu.addressbook.data.person;
+
+public class Block {
```

Comment:
Remember to include header comments for these classes to tell developments how to use these classes.